torchgeo/models/resnet.py

```diff
@@ -108,59 +96,47 @@ class ResNet50_Weights(WeightsEnum):  # type: ignore[misc]
         },
     )
 
-    SENTINEL2_ALL_MOCO = Weights(
-        url=(
-            "https://huggingface.co/torchgeo/resnet50_sentinel2_all_moco/"
-            "resolve/main/resnet50_sentinel2_all_moco.pth"
-        ),
+    SENTINEL2_ALL_DINO = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_sentinel2_all_dino/resolve/main/resnet50_sentinel2_all_dino-d6c330e9.pth",  # noqa: E501
         transforms=_zhu_xlab_transforms,
         meta={
             "dataset": "SSL4EO-S12",
             "in_chans": 13,
             "model": "resnet50",
             "publication": "https://arxiv.org/abs/2211.07044",
             "repo": "https://github.com/zhu-xlab/SSL4EO-S12",
-            "ssl_method": "moco",
+            "ssl_method": "dino",
         },
     )
 
-    SENTINEL2_RGB_MOCO = Weights(
-        url=(
-            "https://huggingface.co/torchgeo/resnet50_sentinel2_rgb_moco/"
-            "resolve/main/resnet50_sentinel2_rgb_moco.pth"
-        ),
+    SENTINEL2_ALL_MOCO = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_sentinel2_all_moco/resolve/main/resnet50_sentinel2_all_moco-df8b932e.pth",  # noqa: E501
         transforms=_zhu_xlab_transforms,
         meta={
             "dataset": "SSL4EO-S12",
-            "in_chans": 3,
+            "in_chans": 13,
             "model": "resnet50",
             "publication": "https://arxiv.org/abs/2211.07044",
             "repo": "https://github.com/zhu-xlab/SSL4EO-S12",
             "ssl_method": "moco",
         },
     )
 
-    SENTINEL2_ALL_DINO = Weights(
-        url=(
-            "https://huggingface.co/torchgeo/resnet50_sentinel2_all_dino/"
-            "resolve/main/resnet50_sentinel2_all_dino.pth"
-        ),
+    SENTINEL2_RGB_MOCO = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_sentinel2_rgb_moco/resolve/main/resnet50_sentinel2_rgb_moco-2b57ba8b.pth",  # noqa: E501
         transforms=_zhu_xlab_transforms,
         meta={
             "dataset": "SSL4EO-S12",
-            "in_chans": 13,
+            "in_chans": 3,
             "model": "resnet50",
             "publication": "https://arxiv.org/abs/2211.07044",
             "repo": "https://github.com/zhu-xlab/SSL4EO-S12",
-            "ssl_method": "dino",
+            "ssl_method": "moco",
         },
     )
 
     SENTINEL2_RGB_SECO = Weights(
```

Comment:
We have defined the `zhu_lab_transforms`, and I was looking into the Seco code again and at least for the bigearthnet datamodule on which they run experiments, augmentations are defined [here](https://github.com/ServiceNow/seasonal-contrast/blob/8285173ec205b64bc3e53b880344dd6c3f79fa7a/datasets/bigearthnet_datamodule.py#L64). Do you want to add those?

Reply:
Yes! Can you add it in a separate PR though? That can fix #1042.